### PR TITLE
Add content for data licence episode

### LINKED ISSUE #7: Add content to episode Data license
The Data license is still a rough outline, inspired by the what and why sections of https://www.howtofair.dk and other resources available in our collaboration document. It needs to be developed further.

### PR DESCRIPTION
**Pull request by Peter9192**
Great work in progress
closes #7

### CODE REVIEW

**Comment by stefsmeets** [comment]
This lesson is mostly complete, except for the Data ownership section.

Also, a key point for the data ownership must be added.

File: _episodes/licenses.md [hunk 1]
```diff
@@ -1,24 +1,149 @@
...
+- A licence cannot be revoked once the data has been shared.
+- As the origial author, you can change the licence for future publications.
+- Sensivite data / embargo
+- The less restrictions you add to your data, the more it can be re-used
+
+Further reading:
+- <https://www.dcc.ac.uk/guidance/how-guides/license-research-data>
+- <https://www.openaire.eu/research-data-how-to-license/>
+
+> ## Excercise
```
Comment (Peter9192) [inline _episodes/licenses.md]:
```suggestion
> ## Discussion
```

File: _episodes/licenses.md [hunk 2]
```diff
@@ -1,24 +1,149 @@
...
+
+In general, data sources in geoscience such as CMIP5 and CMIP6 are available under FAIR licences. That is, you can freely use the data, but there are still some requirements. For example, if you base your research on CMIP6 data, you must give proper credit to the project if you deposit your research data somewhere. This is specified in the Terms of Use.
+
+You can find the Terms of Use for some widely used projects here:
+
+- CMIP6: <https://pcmdi.llnl.gov/CMIP6/TermsOfUse/TermsOfUse6-1.html>
+- CMIP5: <https://pcmdi.llnl.gov/mips/cmip5/terms-of-use.html>
+- CORDEX: <http://is-enes-data.github.io/cordex_terms_of_use.pdf>
+
+>## Exercise
```
Comment (Peter9192) [inline _episodes/licenses.md]:
```suggestion
>## Discussion
```

File: _episodes/licenses.md [hunk 3]
```diff
@@ -1,24 +1,149 @@
...
+>- Where can you find the licence for CMIP6 data? What about CMIP5?
```
Comment (Peter9192) [inline _episodes/licenses.md]:
```suggestion
```
That's mentioned just before the discussion box

File: _episodes/licenses.md [hunk 4]
```diff
@@ -1,24 +1,149 @@
...
+>- How do the terms of use for CMIP6 data differ from the Creative Commons licences?
+>- Can you reshare the data?
+>- Should you attribute the authors of the data? If so, how?
+>- Can derived data be used commecially?
+>- **FIXME**: check what the answer should be
+{: .discussion}
+
+Further reading:
+- https://gmd.copernicus.org/articles/11/3659/2018/ (Section 4)
```
Comment (Peter9192) [inline _episodes/licenses.md]: link doesn't work

File: _episodes/licenses.md (hunk 3 above)
Comment (Peter9192) [reply _episodes/licenses.md]: Perhaps add "Can you find additional terms of licences that are applicable to your situation?"

File: _episodes/licenses.md [hunk 5]
```diff
@@ -1,24 +1,149 @@
...
+You may be restricted by your institutions’ policies on licenses
+
+Data ownership is defined in the data management plan
+
+
+> ## Excercise
+>Take a few minutes to find out whether your institute has a license policy.
+>
+>- What does it say?
+>- Who owns *your* data?
```
Comment (Peter9192) [inline _episodes/licenses.md:103]: What to do if your institution doesn't have a policy?

File: _episodes/licenses.md [hunk 6]
```diff
@@ -1,24 +1,149 @@
...
+{: .discussion}
+
+> ## Excercise
+>- Imagine you have downloaded some data from source X and applied operation Y on it.
```
Comment (Peter9192) [inline _episodes/licenses.md]: FIXME

File: _episodes/licenses.md [hunk 7]
```diff
@@ -1,24 +1,149 @@
...
+
+How the data will be licenced is part of the data management plan. Specificically:
+
+- How will the data be licensed to permit the widest re-use possible?
+- When will the data be made available for re-use (i.e. are the data under embargo)?
+- Are the data usable by third parties (i.e. are there restrictions)?
+
+In all cases, the Commission encourages researchers to provide access to their data in the broadest sense. They encourage authors to retain their copyright, but grant adequate licences to publishers. They recommend the Creative Common licences as a useful legal tool for providing open access.
+
+Further reading:
```
Comment (Peter9192) [inline _episodes/licenses.md:138]: Perhaps we can create an exercise out of this? Let them read the document and find the info themselves? In my experience it is easier to memorize "where to find info" than to memorize the actual info

File: _episodes/licenses.md [hunk 8]
```diff
@@ -1,24 +1,149 @@
...
+Adding a licence to data is easy. You just need to mention or refer to the licence in the same place where you upload your data. Some repositories like Zenodo allow you to select a licence, and they make sure it is clear under which licence the data are available.
 
-License restrictions (share alike etc.) for derived data
+You can apply a licence by one of these ways:
 
-Institutions’ policies on licenses
+- Choosing a licence when you upload your data to a repository
+- Referring to the licence on the webpage where the data are hosted
+- Attaching the licence to the metadata
+- Adding a readme file / licence file to your data
```
Comment (Peter9192) [inline _episodes/licenses.md:153]: Note: some projects (e.g. CMIP6) explicitly require you to add the licence to the metadata.

File: _episodes/licenses.md [hunk 9]
```diff
@@ -1,24 +1,149 @@
...
+Further reading:
+- <https://ec.europa.eu/research/participants/docs/h2020-funding-guide/cross-cutting-issues/open-access-data-management/open-access_en.htm>
+- <https://ec.europa.eu/research/participants/data/ref/h2020/other/hi/oa-pilot/h2020-infograph-open-research-data_en.pdf>
+- In case you are really interested, have a look at [p251-253 of the Horizon2020 Programme](https://ec.europa.eu/research/participants/data/ref/h2020/grants_manual/amga/h2020-amga_en.pdf)
+
+How are licences applied to research data?
+------------------------------------------
 
-CMIP5 and CMIP6 licenses (special case, example)
+Adding a licence to data is easy. You just need to mention or refer to the licence in the same place where you upload your data. Some repositories like Zenodo allow you to select a licence, and they make sure it is clear under which licence the data are available.
```
Comment (Peter9192) [inline _episodes/licenses.md]:
```suggestion
In principle, a licence is valid when it is mentioned or referred to in the same place where you upload your data. Some repositories like Zenodo allow you to select a licence, and they make sure it is clear under which licence the data are available.
```

**Review by Peter9192 — COMMENTED** [review]
Amazing content! 
Couple of small suggestions, and maybe some items for follow-up. Could you either add FIXMEs/TODO's or make new issues to follow up on these? Then we don't forget about them.

**Comment by Peter9192** [comment]
Issue for follow-up on the exercises:  #17

@stefsmeets thanks for the great work. Since I made this PR originally, I cannot approve and merge. Can you do that now?

**Review by stefsmeets — APPROVED** [review]
Beautiful work! 👍

**Pull request merged**